Reference/Searching/Examine/examine-events.md

```diff
@@ -146,3 +146,60 @@ At this point you can create a query for only that field:
 ```c#
 var results = searcher.CreateQuery("content").Field("combinedField", searchTerm).Execute();
 ```
+
+### Adding the path of the node as a searchable field into the index
+
+Sometimes you might want to search descendants of a node rather than immediate children for a particular search term. In this instance, we need to write out the `path` property as a searchable field into the index. The `path` property is a comma-separated string. Due to the presence of the comma in the path, the field is not tokenized therefore it is not searchable. We will inject new
+field with the path that has a comma removed this making it searchable. This can be done by adding the new field using to `IndexProviderTransformingIndexValues` method. So continuing from the above example the method will look like below.
+
+```csharp
+if (e.ValueSet.Category == IndexTypes.Content)
+{
+    var combinedFields = new StringBuilder();
+
+    var pathBuilder = new StringBuilder();
+
+    foreach (var fieldValues in e.ValueSet.Values)
+    {
+        foreach (var value in fieldValues.Value)
+        {
+            if (value != null)
+            {
+                combinedFields.AppendLine(value.ToString());
+            }
+        }
+    }
+
+    if (fieldValues.Key == "path")
+    {
+        foreach (var value in fieldValues.Value)
+        {
+            if (value != null)
+            {
+                var path = value.ToString().Split(',');
```

Comment:
var path = value.ToString().Replace(",", " ");
or
string.Join(" ", path);
??
instead of doing the foreach?